tests/test_space_ops.py

```diff
@@ -0,0 +1,10 @@
+import numpy as np
+
+from manim import cartesian_to_spherical, spherical_to_cartesian
+
+
+def test_polar_coords():
+    a = np.array([1, 1, 0])
+    b = (2, np.pi / 2, np.pi / 2)
+    assert cartesian_to_spherical(a) == (2 ** 0.5, np.pi / 2, np.pi / 4)
```

Comment:
I think you should round here too.

Reply:
I think this is unnecessary, I would prefer not rounding since it doesn't have rounding issues.

Reply:
It could break on different systems. I'd definitely prefer rounding. This was done for the font size test [here](https://github.com/ManimCommunity/manim/pull/1694#discussion_r665212160), even though the number of decimal places didn't cause a problem there either.